### PR TITLE
always group ip block

### PR DESCRIPTION
**Pull request by ShiriMoran**

### CODE REVIEW

File: pkg/ibmvpc/examples/acl_testing3.txt [hunk 1]
```diff
@@ -1,55 +1,14 @@
...
-172.32.0.0/11 => vsi2-ky[10.240.20.4] : All Connections
-192.0.1.0/24 => vsi2-ky[10.240.20.4] : All Connections
-192.0.3.0/24 => vsi2-ky[10.240.20.4] : All Connections
-192.169.0.0/16 => vsi2-ky[10.240.20.4] : All Connections
-192.88.100.0/22 => vsi2-ky[10.240.20.4] : All Connections
-198.20.0.0/14 => vsi2-ky[10.240.20.4] : All Connections
-198.51.101.0/24 => vsi2-ky[10.240.20.4] : All Connections
-203.0.114.0/23 => vsi2-ky[10.240.20.4] : All Connections
-vsi1-ky[10.240.10.4] => 161.26.0.0/16 : protocol: UDP *
+1.0.0.0/8,100.128.0.0/9,11.0.0.0/8,128.0.0.0/5,142.0.0.0/8,143.0.0.0/8,147.235.219.206/32,147.235.219.207/32,147.235.219.208/28,161.26.0.0/16,161.27.0.0/16,169.255.0.0/16,172.32.0.0/11,192.0.1.0/24,192.0.3.0/24,192.169.0.0/16,192.88.100.0/22,198.20.0.0/14,198.51.101.0/24,203.0.114.0/23 => vsi2-ky[10.240.20.4] : All Connections
```
Comment (adisos) [inline pkg/ibmvpc/examples/acl_testing3.txt:1]: why isn't this specified as `Public Internet (all ranges) => vsi2-ky[10.240.20.4] : All Connections` , similar to the grouped expected output in https://github.com/np-guard/vpc-network-config-analyzer/blob/main/pkg/ibmvpc/examples/acl_testing3_with_grouping.txt ?

Comment (ShiriMoran) [reply pkg/ibmvpc/examples/acl_testing3.txt:1]: Because mistakenly I did not use the latest main. Now merged. Let me take a look

**Comment by adisos** [comment]
when running with output format `md` the output does not include that grouping.
`$ ./bin/vpcanalyzer -vpc-config pkg/ibmvpc/examples/input_acl_testing3.json -format md`

when running with grouping flag I do get that grouping
` ./bin/vpcanalyzer -vpc-config pkg/ibmvpc/examples/input_acl_testing3.json -format md -grouping`

**Comment by ShiriMoran** [comment]
replaced by https://github.com/np-guard/vpc-network-config-analyzer/pull/128

**Pull request closed without merging**